From 591766401c9df5244a2bae62c7628ae10fb5a6c9 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Thu, 9 Feb 2017 16:34:29 +0900
Subject: [PATCH 1747/2066] arm64: dts: r8a7795-es1: Remove clocks and
 power-domains from MFIS Lock device node

This patch removes clocks and power-domains from MFIS Lock device node.
MFIS clock is always enabled on R-Car Gen3, and driver does not control
the clock.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index 3e1b1e7..fad2a0f 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -814,8 +814,6 @@
 			compatible =    "renesas,mfis-lock-r8a7795",
 					"renesas,mfis-lock";
 			reg = <0 0xe62600c0 0 0x0020>;
-			clocks = <&cpg CPG_MOD 213>;
-			power-domains = <&sysc R8A7795_PD_ALWAYS_ON>;
 			status = "okay";
 		};
 
-- 
1.7.5.4

